packages/base/tests/blockchain.test.ts

```diff
@@ -99,3 +99,51 @@ test("Script codec", (t) => {
     "0x370000001000000030000000310000009bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce801020000000000"
   );
 });
+
+test("header codec", (t) => {
```

Comment:
would you please provide where the test data from

Reply:
dummy data I created by myself